src/interfaces.ts

```diff
@@ -20,7 +20,8 @@ interface IFieldConfigBase {
   editProps: { [key: string]: any };
   field: string;
   fieldConfigProp: boolean;
-  formItemProps?: { [key: string]: any };
+  formItemProps?: Partial<FormItemProps>;
+  formItemRenderExtra?: (value: IValue) => IValue;
```

Comment:
Returning `IValue` is a little vague here. Wouldn't we expect it to be a string or react node?